engine/src/main/java/org/terasology/rendering/dag/tasks/SetFacesToCullTask.java

```diff
@@ -22,10 +22,10 @@
 /**
  * TODO: Add javadocs
  */
-public final class SetWhatFacesToCullTask implements RenderPipelineTask {
+public final class SetFacesToCullTask implements RenderPipelineTask {
```

Comment:
Let's add:

// TODO: review the possibility of making task classes private, contained in the StateChange implementation they are associated with. Rationale: module developers shouldn't be able to reach or instantiate these classes directly: only the StateChange classes.

What do you think?